src/NetPanzer/Views/Components/newButton.hpp

```diff
@@ -37,15 +39,15 @@ class newButton : public Button {
     label = Title;
     setTextColors(Color::white, Color::lightYellow, Color::darkGray);
     setExtraBorder();
-    borders[0][0] = Color::gray;
-    borders[0][1] = Color::black;
-    borders[1][0] = Color::gray;
-    borders[1][1] = Color::gray;
-    borders[2][0] = Color::gray;
-    borders[2][1] = Color::darkGray;
+    borders[0][0] = MenuConfig::button_normal_border;
+    borders[0][1] = MenuConfig::button_normal_highlight;
+    borders[1][0] = MenuConfig::button_hover_border;
+    borders[1][1] = MenuConfig::button_hover_highlight;
+    borders[2][0] = MenuConfig::button_pressed_border;
+    borders[2][1] = MenuConfig::button_pressed_highlight;
     Surface bnormal;
-    bnormal.loadBMP("pics/backgrounds/menus/buttons/default/page_green.bmp");
-    Surface spbutton(Surface::getTextLength(Title) + 20, 25, 1);
+    bnormal.loadBMP(("pics/backgrounds/menus/buttons/default/" + *MenuConfig::button_texture).c_str());
+    Surface spbutton(Surface::getTextLength(Title) + 20, MenuConfig::menu_font_size > 25 ? MenuConfig::menu_font_size : 25, 1);
```

Comment:
this is basically min(font_size, 25) right? any reason to do this?

Reply:
it is. I'm bad at cpp and couldn't find the min function. I think I set it like this because I didn't want the buttons to shrink below a height of 25. I will look at this code again (and add a comment).

Reply:
yeah probably good enough to just have a comment in lua "beyond these ranges it doesn't work" :P